github/util.py

```diff
@@ -95,6 +95,19 @@ def create_or_update_file(
 
         return response["commit"].sha
 
+    def retrieve_text_file_contents(
+        self,
+        repository_branch: str,
+        repository_path: str,
+        encoding: str = 'utf-8',
```

Comment:
still no spaces around the equals character ('=') according to PEP-8 - this did not change during the past few weeks, afaik.